Dockerfile

```diff
@@ -1 +1,32 @@
-FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/base_centos:7-20210630
+FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/centos/base-image:java8
+
+ENV SOLR_VERSION 7.7.2
+
+ENV SOLR_URL https://archive.apache.org/dist/lucene/solr/$SOLR_VERSION/solr-$SOLR_VERSION.tgz
+
+ENV SOLR_PORT 8983
+
+ADD $SOLR_URL /tmp
+
+RUN useradd -M solr && \
+    cd /tmp && \
+    yum install lsof -y && \
```

Comment:
Please leave a comment inside the Dockerfile as to why `lsof` is needed.

Reply:
I can't see a comment in the Dockerfile explaining why `lsof` is needed. Please leave a short comment explaining why `lsof` is needed by the containerized app.